tests/fsharp/Compiler/ErrorMessages/SuggestionsTests.fs

```diff
@@ -0,0 +1,321 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.UnitTests
+
+open NUnit.Framework
+open FSharp.Compiler.SourceCodeServices
+
+[<TestFixture>]
+module Suggestions =
+
+    [<Test>]
+    let ``Field Suggestion`` () =
+        CompilerAssert.TypeCheckSingleError
+            """
+type Person = { Name : string; }
+
+let x = { Person.Names = "Isaac" }
+            """
+            FSharpErrorSeverity.Error
+            39
+            (4, 18, 4, 23)
+            "The field, constructor or member 'Names' is not defined."
```

Comment:
Do we not care to verify the suggestion ?

````
// #Warnings
//<Expects status="Error" id="FS0039"></Expects>
//<Expects>Maybe you want one of the following:</Expects>
//<Expects>Name</Expects>
````

Reply:
Ouch. I think I now understand @MecuSorin's questions in the email. It looks like we don't add the "suggest names" flag in compilerassert. This is of course not really what we want.
Sorry for the confusion mate. I didn't realize you meant the suggested names. I just thought we were talking about other discrepancies in the error messages that happen because the fsharpqa is using regex and this one is not. 

So we actually need to set the correct flag in compilerassert (I do that) and then change the error messages back. Sorry for double work. This one is really my fault

Reply:
No problem. Just let me know what I should do. I think that I can recover the old version with git reflog, somehow I hit that notorious git blackhole last night and my attempt to fix it by moving my master to the last commit in github PR commits list - made PR virtually disappear ::facepalm .

Reply:
Must be something like https://github.com/dotnet/fsharp/pull/7251 - but can't verify right now.

Reply:
I suggest to accept this PR as it is, and to come back to it (add suggestions to tests) when the mechanism for verify suggested names will be available.

Reply:
@mecusorin I suggest we keep it open for now and I will send a PR to your fork soon. I don't want to lose suggestion tests in master. It doesn't hurt to keep it open for couple of days right?

Reply:
Can you please merge https://github.com/dotnet/fsharp/pull/7251?

Reply:
Trust me I like too much F# to suggest losing those checks (for suggested names).

Let me suggest an alternative: I think that we better merge this as it is and do the rest of the work in the #7251 - I will make PR in your repo (with the names check) so you can carry them as a single change (from semantic perspective).

Let me know if you still prefer merging the work in #7251 (A) or my suggestion to do it in 2 separate steps (B)

I prefer the B because at the moment I still cannot run the tests (waiting for a mobo next week, then I will be able to make a fresh install, including latest VS) and I am afraid that the change will trigger failing tests in other places too.

Reply:
@forki, it looks like there are still test issues with #7205 

@mecusorin, it's fine for this to hang around until #7205 is completed and merged.

We are not in a huge rush :-)

Kevin